flink-yarn/src/main/java/org/apache/flink/yarn/YarnClusterDescriptor.java

```diff
@@ -911,31 +929,21 @@ private ApplicationReport startAppMaster(
         final List<String> systemClassPaths = fileUploader.registerProvidedLocalResources();
         final List<String> uploadedDependencies =
                 fileUploader.registerMultipleLocalResources(
-                        systemShipFiles.stream()
-                                .map(e -> new Path(e.toURI()))
-                                .collect(Collectors.toSet()),
-                        Path.CUR_DIR,
-                        LocalResourceType.FILE);
+                        systemShipFiles, Path.CUR_DIR, LocalResourceType.FILE);
```

Comment:
`systemShipFiles` includes 3 types of files:

- ShipFiles
- logConfigFilePath
- LibFolders

After this PR, the latter two types do not add schema, does it work as expected?

Reply:
@1996fanrui Thank you for your view.    
The LogConfigFilePath works in my Flink environment.  
Then I used `new Path(file.toRUI())` when adding `logConfigFilePath` and `LibFolders` to `systemShipFiles`. After the change, the `logConfigFilePath` also worked. Maybe it's a better choice to handler the 3 types of files in the same way.

Reply:
I have tested both method to create `new Path()` can work for `LibFolders`.